projects/shared/src/lib/view/thumbnail/thumbnail.component.ts

```diff
@@ -78,9 +78,13 @@ export class ThumbnailComponent implements OnInit {
    */
   ngOnInit() {
     if (this.record && this.record.metadata) {
-      this.coverUrl = `/static/images/icon_${this.record.metadata.type}.png`;
-      if (this.record.metadata.cover_art) {
-        this.coverUrl = this.record.metadata.cover_art;
+      this.coverUrl = `/static/images/icon_${this.record.metadata.type[0].main_type}.svg`;
+      if (this.record.metadata.electronicLocator) {
+        for (const electronicLocator of this.record.metadata.electronicLocator) {
+          if (electronicLocator.content === 'coverImage' && electronicLocator.type === 'relatedResource') {
+            this.coverUrl = electronicLocator.url;
```

Comment:
add a `break` statement to stop the loop if a cover url is found.

Reply:
ok, done

Reply:
ok, done